### PR TITLE
web_company_logo_unclickable module

### PR DESCRIPTION
**Pull request by rami-wafaie**

### CODE REVIEW

**Comment by ntodorova** [comment]
:+1:

File: web_company_logo_unclickable/static/src/xml/base.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<templates id="template" xml:space="preserve">
+    <t t-extend="WebClient">
+        <t t-jquery="a[class=oe_logo]" t-operation="replace">
```
Comment (hparfr) [inline web_company_logo_unclickable/static/src/xml/base.xml:5]: Why not just remove the _src_ attribute of **a** ?

Comment (hbrunn) [inline web_company_logo_unclickable/static/src/xml/base.xml:5]: your mean the `href` attribute, but yes, I think that's much cleaner

Comment (hparfr) [inline web_company_logo_unclickable/static/src/xml/base.xml:5]: Correct, I mean href instead of src. ;-)

Comment (rami-wafaie) [inline web_company_logo_unclickable/static/src/xml/base.xml:5]: I tried to, but I need to keep "Edit Company Data" the way it is, just remove the clickable link from the logo, so any ideas to make it better?

Comment (hbrunn) [inline web_company_logo_unclickable/static/src/xml/base.xml:5]: untested from the top off my head:

``` xml
<t t-jquery="a[class=oe_logo]">
jQuery(this).removeAttr('href');
</t>
```

**Pull request closed without merging**